addon/fragment.js

```diff
@@ -159,13 +159,15 @@ const Fragment = Model.extend(Ember.Comparable, Copyable, {
  * @param {Object} data the fragment data
  * @return {String} the actual fragment type
  */
-export function getActualFragmentType(declaredType, options, data) {
+export function getActualFragmentType(declaredType, options, data, owner) {
   if (!options.polymorphic || !data) {
     return declaredType;
   }
 
   let typeKey = options.typeKey || 'type';
-  let actualType = data[typeKey];
+  console.log('data', data);
+  console.log('owner', owner);
```

Comment:
Looks like forgotten console logs

Reply:
Oh my god sorry x). I must be tired ahah